pom.xml

```diff
@@ -101,6 +101,9 @@
         <configuration>
           <workingDirectory>${project.basedir}/src/main/webapp</workingDirectory>
           <installDirectory>target</installDirectory>
+          <environmentVariables>
+            <BACKEND_URL>${env.BACKEND_URL}</BACKEND_URL>
```

Comment:
please put the `export BACKEND_URL=http://localhost:8080` also into the README.md